packages/grafana-ui/src/components/TimeSeries/TimeSeries.tsx

```diff
@@ -18,8 +18,15 @@ export class UnthemedTimeSeries extends React.Component<TimeSeriesProps> {
 
   prepConfig = (alignedFrame: DataFrame, getTimeRange: () => TimeRange) => {
     const { eventBus, sync } = this.context;
-    const { theme, timeZone } = this.props;
-    return preparePlotConfigBuilder({ frame: alignedFrame, theme, timeZone, getTimeRange, eventBus, sync });
+    const { theme } = this.props;
+    return preparePlotConfigBuilder({
+      frame: alignedFrame,
+      theme,
+      timeZone: this.panelContext.timeZone,
```

Comment:
why `this.panelContext` while you have te context available via `this.context`? I think the entire panelContext property is not necessary.

Reply:
because I am learning to use context!